src/Commands/Foundry/Models/ModelGuidanceCommand.cs

```diff
@@ -0,0 +1,77 @@
+using System.CommandLine;
+using System.CommandLine.Parsing;
+using AzureMcp.Models.Command;
+using AzureMcp.Services.Interfaces;
+using ModelContextProtocol.Server;
+using AzureMcp.Arguments.Foundry.Models;
+using AzureMcp.Models.Argument;
+using AzureMcp.Arguments.Cosmos;
+
+namespace AzureMcp.Commands.Foundry.Models;
+
+public sealed class ModelGuidanceCommand : GlobalCommand<ModelGuidanceArguments>
+{
+    private readonly Option<string> _inferanceNameOption = ArgumentDefinitions.Foundry.InferenceModelName.ToOption();
+    protected override string GetCommandName() => "guidance";
```

Comment:
Please rebase with main.  We update the BaseCommand interface. Make sure you add the title property as well and update the McpServerTool attribute with the title.